authors/apps/articles/views.py

```diff
@@ -295,26 +296,37 @@ def delete(self, request, pk, format=None):
 
 class LikesView(APIView):
     """
-    View for liking and disliking Articles and later on Comments
+    View for liking and disliking Articles and Comments
     """
     model = None  # Data Model - Articles or Comments
     vote_type = None  # Vote type Like/Dislike
     permission_classes = (IsAuthenticatedOrReadOnly, IsVerifiedUser,)
 
     def post(self, request, *args, **kwargs):
         """
-        Method to like or dislike an article
+        Method to like or dislike an article or a comment
 
         Params
         -------
         request: Object with request data and functions.
-        kwargs: reference to the article to liked/disliked
+        kwargs: reference to the article/comment to be liked/disliked
 
         Returns
         --------
-        total number of likes and dislikes for that article
-        """
-        obj = self.model.objects.get(slug=self.kwargs['slug'])
+        total number of likes and dislikes for that article/comment
+        """
+        if self.model == Articles:
```

Comment:
Instead of having this `if block` and multiple `try..` blocks, which just adds to code complexity, you could make use of a mapper, eg:
```
        model_mapper = {
            Comment: ({'id': self.kwargs.get('id', '')}, CommentNotFound),
            Articles: ({'slug': self.kwargs.get('slug', '')}, ArticleNotFound),
        }
        try:
            obj = self.model.objects.get(**model_mapper[self.model][0])
        except self.model.DoesNotExist:
            raise model_mapper[self.model][1]
```
You could also make use of this for the `send_notifications` part